modules/core/core-scheduler/src/main/java/com/enonic/xp/impl/scheduler/distributed/CronCalendarImpl.java

```diff
@@ -17,11 +17,11 @@
 import com.cronutils.parser.CronParser;
 import com.google.common.base.Preconditions;
 
-import com.enonic.xp.scheduler.ScheduleCalendar;
+import com.enonic.xp.scheduler.CronCalendar;
 import com.enonic.xp.scheduler.ScheduleCalendarType;
 
-public final class CronCalendar
-    implements ScheduleCalendar
+public final class CronCalendarImpl
```

Comment:
to avoid confusion such Impl classes should be package local.
They should be other OSGi classloaders but nothing but this module should be able to instantiate them.